.gitignore

```diff
@@ -23,3 +25,7 @@ yarn-debug.log*
 yarn-error.log*
 
 test-report.xml
+*.tgz
+dist/lib.js
```

Comment:
Since we already ignore `/dist` above, we don't need this `dist/lib.js` entry too.

Reply:
👍